### ISSUE TITLE
I don't understand how to calculate pair_ wise_ cls_ loss

### ISSUE THREAD

**Issue by xiaowuge1201** [Megvii-BaseDetection/YOLOX#436]
pair_wise_cls_loss = F.binary_cross_entropy(
            cls_preds_.sqrt_(), gt_cls_per_image, reduction="none"
        ).sum(-1)
cls_preds_.sqrt_()？？？？？？Why CLS_ preds_ take a square root？？？？

**Comment by maoyichun** [comment]
@xiaowuge1201 同样不理解这个地方，有思路吗？
